File: launchdarkly/typescript-node/src/GetUserFlagSettingExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.UserSettingsApi();
apiCaller.setApiKey(api.UserSettingsApiApiKeys.ApiKey, "YOUR_API_KEY");

apiCaller.getUserFlagSetting(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  "userKey_string", // userKey
  "featureFlagKey_string", // featureFlagKey
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling UserSettingsApi#getUserFlagSetting:");
  console.log(error.body);
});

File: launchdarkly/kotlin/src/main/kotlin/oseg/launchdarkly_examples/GetUserFlagSettingExample.kt
package oseg.launchdarkly_examples

import com.launchdarkly.client.infrastructure.*
import com.launchdarkly.client.apis.*
import com.launchdarkly.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class GetUserFlagSettingExample
{
    fun getUserFlagSetting()
    {
        ApiClient.apiKey["Authorization"] = "YOUR_API_KEY"

        try
        {
            val response = UserSettingsApi().getUserFlagSetting(
                projectKey = "projectKey_string",
                environmentKey = "environmentKey_string",
                userKey = "userKey_string",
                featureFlagKey = "featureFlagKey_string",
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling UserSettingsApi#getUserFlagSetting")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling UserSettingsApi#getUserFlagSetting")
            e.printStackTrace()
        }
    }
}
